backend/corpora/common/entities/dataset.py

```diff
@@ -169,12 +151,10 @@ def asset_deletion(self):
             asset.delete_from_s3()
             asset.delete()
 
-    def deployment_directories_deletion(self):
-        for deployment_directory in self.deployment_directories:
-            object_names = get_cxg_bucket_path(deployment_directory)
-            logger.info(f"Deleting all files in bucket {cxg_bucket.name} under {object_names}.")
-            cxg_bucket.objects.filter(Prefix=object_names).delete()
-            self.session.delete(deployment_directory)
+    def delete_explorer_cxg_object_from_s3(self):
```

Comment:
Wouldn't you also want to clear the two fields `explorer_s3_uri` and `explorer_url` here as well if you were to delete the artifact which can occur beyond deleting a dataset (say the conversion of CXG failed).

Reply:
This function is intended to only delete from s3. This is because of the requirement to not delete resources from S3 until after the changes are published.

Reply:
Got it, thanks!